src/main/java/uk/gov/hmcts/ccd/data/casedetails/DefaultCaseDetailsRepository.java

```diff
@@ -207,16 +207,16 @@ private Query getQueryByParameters(MetaData metadata, final Map<String, String>
     }
 
     private Query getCountQueryByMetaData(MetaData metadata) {
-        return queryBuilderFactory.count(em)
-                                  .whereMetadata(metadata)
-                                  .build();
+        return queryBuilderFactory.count(em, metadata)
+            .whereMetadata(metadata)
+            .build();
```

Comment:
The 2 lines above should be indented and aligned with `.count`

Reply:
indented